sdk/core/azure-core/src/environment_log_level_listener.cpp

```diff
@@ -125,7 +125,8 @@ EnvironmentLogLevelListener::GetLogListener()
                   << Azure::DateTime(std::chrono::system_clock::now())
                          .ToString(
                              DateTime::DateFormat::Rfc3339, DateTime::TimeFractionFormat::AllDigits)
-                  << "] " << LogLevelToConsoleString(level) << " : " << message;
+                  << " T: " << std::hex << std::this_thread::get_id() << std::dec << "] "
```

Comment:
Would it be better to be structured
[DateTime][ThreadId] <message?

Reply:
Maybe? I was operating on a model of "the stuff within the [] is metadata about the origin of the log", which in this case is the time of the log and the thread which generated the log.

But I could see it being two separate unrelated items.

But IMHO adding the "T:" is critical - the date-time element is self explanatory, but the thread ID is not at all self-explanatory.